workshops/templates/workshops/person_bulk_add_form.html

```diff
@@ -1,13 +1,10 @@
 {% extends "base_nav_fixed.html" %}
 
 {% block content %}
+{% load crispy_forms_tags %}
 {% load staticfiles %}
+
 <p><a href="{% url 'person_bulk_add_template' %}">BulkPersonAddTemplate.csv</a> is a blank template of a well-formed .CSV file.</p>
-<form action="" method="post" enctype="multipart/form-data"
```

Comment:
Do we have to show that the "import using the URL field" is a functionality exclusive to PyData events?
But since PyData will be hosting a separate instance of AMY, this shouldn't be a problem.